### DISCUSSION TITLE
Omnichannel - New Chat welcome message?

### DISCUSSION THREAD

**Discussion by git-tec** [RocketChat/Rocket.Chat#29665] [Ideas]
I also find this a very important feature. After entering your data as a chat user (omnichannel), there is only an empty chat.

There is definitely a custom welcome message missing here, which for example tells the participant to ask their question and submit it.

Attached is an example from our old live chat software Livezilla.

![beispiel_livezilla](https://github.com/RocketChat/Rocket.Chat/assets/136434874/a726d255-cfb6-4064-be40-f76c3aaf4c21)

**Comment by MartinSchoeler** [comment]
Hey, you could probably achieve something similar using [Livechat triggers](https://docs.rocket.chat/use-rocket.chat/omnichannel/livechat-triggers)

**Reply by git-tec** [reply]
Thanks, but this works only partially, if you start the livechat in popout mode (/livechat?mode=popout) it does not work.
